api/lib/infrastructure/repositories/knowledge-element-repository.js

```diff
@@ -179,7 +179,7 @@ module.exports = {
       const validatedTargetedKnowledgeElements = _filterValidatedTargetedKnowledgeElements(knowledgeElements, targetProfile);
       const knowledgeElementsByCompetenceId = _.groupBy(validatedTargetedKnowledgeElements, 'competenceId');
       knowledgeElementsGroupedByUserAndCompetence[userId] = {};
-      for (const competenceId  of competencesIds) {
+      for (const competenceId of competencesIds) {
```

Comment:
[suggestion] Pour faire comme tu as fait dans la fonction suivante
```suggestion
      for (const competenceId of competenceIds) {
```

Reply:
bieng vu